cfgov/unprocessed/css/molecules/pagination.less

```diff
@@ -0,0 +1,158 @@
+/* topdoc
+  name: Pagination
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+          <nav class="m-pagination">
+              <a class="btn btn__super m-pagination_btn_prev" href="#o-pagination-content_list">
+                  <span class="btn_icon__left cf-icon cf-icon-left"></span>
+                  Previous
+              </a>
+              <a class="btn btn__super m-pagination_btn_next" href="#o-pagination-content_list">
+                  Next
+                  <span class="btn_icon__right cf-icon cf-icon-right"></span>
+              </a>
+              <form data-js-hook="validate_pagination" action="#o-pagination-content_list">
+                  <label for="m-pagination_current-page">
+                      Page
+                      <span class="u-visually-hidden">
+                      </span>
+                  </label>
+                  <input id="m-pagination_current-page"
+                         name="m-pagination_current-page"
+                         type="number" min="" max=""
+                         value="">
+                  <span class="m-pagination_label">
+                      <span aria-hidden="true">
+                      </span>
+                  </span>
+                  <button id="m-pagination_submit"
+                          type="submit">
+                      Go
+                  </button>
+              </form>
+          </nav>
+    codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-pagination
+            .btn btn__super.m-pagination_btn_prev
+            .btn btn__super.m-pagination_btn_next
+          form
+            label
+              .u-visually-hidden
+            input
+            .m-pagination_label
+              span
+            button
+  tags:
+    - cfgov-molecules
+*/
+
+@pagination-bg-color: #F0F1F1;
+@pagination-text-color: #75787B;
+@pagination-font-size__px: 16px;
+@pagination-font-size__em: unit( @pagination-font-size__px / @base-font-size-px, em );
+@pagination-height__px: 42px;
+@pagination-width__px: 130px;
+
+.m-pagination {
+    position: relative;
```

Comment:
`min-width: 265px` or equivalent could be added here to avoid overlapping of the buttons at very small sizes. But perhaps that's too small to be necessary.